code/game/atoms.dm

```diff
@@ -362,21 +362,32 @@
 					if(T in shuttle_area)
 						return TRUE
 
-	if(!is_centcom_level(T.z))//if not, don't bother
-		return FALSE
+	if(is_centcom_level(T.z))
+		return TRUE
```

Comment:
Mobs in the ninja dojo or abductor area really shouldn't count as escaped though